servicecontrol/troubleshooting.md

```diff
@@ -44,5 +43,8 @@ netsh http add urlacl url=http://localhost:33333/api/ user=<accountname> Listen=
     * `%WINDIR%\System32\config\systemprofile\AppData\Local\Particular\ServiceControl\logs` if the issue relates to ServiceControl normal operations. Logs location may vary depending on the user that has been configured to run the ServiceControl service, the above one is the one where the LocalSystem user outputs logs information;
 * Most common cause is prerequisites installation and configuration issues;
 
- 
+### Particular.ServiceControl fails to start: `EsentInstanceUnavailableException`
+
+If the Particular.ServiceControl fails to start raising in the logs a `Microsoft.Isam.Esent.Interop.EsentInstanceUnavailableException` ensure that ServiceControl database directory, sub-directory and files, is excluded from any anti-virus and anti-maleware real-time and scheduled scan.
 
+NOTE: ServiceControl default database directory is located at `%SystemDrive%\ProgramData\Particular\ServiceControl\localhost-33333`
```

Comment:
fixed, or maybe call it out as a NOTE is better?

On Mon, Apr 20, 2015 at 1:45 PM, Simon Cropp <notifications@github.com>
wrote:

> In servicecontrol/troubleshooting.md
> <https://github.com/Particular/docs.particular.net/pull/600#discussion_r28681893>
> :
>
> >
> > +NOTE: ServiceControl default database directory is located at `%SystemDrive%\ProgramData\Particular\ServiceControl\localhost-33333`
>
> should prob just link to here
> http://docs.particular.net/servicecontrol/configure-ravendb-location
> instead of mentioning the location again here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/600/files#r28681893>
> .
>



-- 
Mauro Servienti | Solution Architect | Particular Software